atlp-rwanda/elite-bn-fe
# n

#### What does this PR do?

Setting up the tests

#### Description of Task to be completed?

The main objective of this chore is to isolate written code to test and determine if it works as intended.

#### How should this be manually tested?

pre-requisite install node and yarn make sure you have the updated versions.

Initial clone the repo

run : ` yarn install `

afterwards run:  `yarn test` to run the test using Jest

#### Any background context you want to provide?

https://reactjs.org/docs/testing-recipes.html for documentation on react unit testing 
https://jestjs.io/ for documentation on jest


#### What are the relevant pivotal tracker stories?

https://www.pivotaltracker.com/story/show/182201517

#### Screenshots (if appropriate)
<img width="1440" alt="Screen Shot 2022-07-27 at 00 12 28" src="https://user-images.githubusercontent.com/71593401/181121309-318f1d47-92cd-44a3-8a3c-6e60bfdbf29f.png">
